warehouse/utils/utils.go

```diff
@@ -305,6 +305,20 @@ type TriggerUploadRequest struct {
 	DestinationID string `json:"destination_id"`
 }
 
+type Connection struct {
+	SourceId      string   `json:"source_id"`
+	DestinationId string   `json:"destination_id"`
```

Comment:
minor: for go idiomatic reasons and consistency
```suggestion
	SourceID      string   `json:"source_id"`
	DestinationID string   `json:"destination_id"`
```